### PR TITLE
BAVL-379 initial commit for enabling the new BVLS service in whereabouts-api

### PR DESCRIPTION
**Pull request by matt-crowson-moj**

### CODE REVIEW

File: migration/rollout_new_bvls_service.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		break;;
+	[nN] ) echo exiting...;
+		exit;;
+	* ) echo invalid response;;
+esac
+
+done
+
+# Each service that needs changes applying should be added as a separate file and sourced below.
+source ./whereabouts-api.sh "$ENVIRONMENT"
```
Comment (matt-crowson-moj) [inline migration/rollout_new_bvls_service.sh:29]: Plan here is we can just add the other namespaces as separate files.  Just need to pass the `$ENVIRONMENT` through.

File: migration/whereabouts-api.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  LISTEN_FOR_COURT_EVENTS: "false"
+  SEND_VIDEO_BOOKING_MIGRATE_EVENTS: "true"
+EOF
+
+# Apply the secret
+kubectl apply -f ./$WHEREABOUTS_SECRETS_FILE
+echo "Applied secrets to $WHEREABOUTS_NAMESPACE"
+
+kubectl -n "$WHEREABOUTS_NAMESPACE" get secrets feature-toggles -o json | jq -r ".data | map_values(@base64d)"
+
```
Comment (timharrison-moj) [inline migration/whereabouts-api.sh:26]: should be singular? `secrets`

**Review by timharrison-moj — APPROVED** [review]
LGTM

File: migration/whereabouts-api.sh (hunk 2 above)
Comment (matt-crowson-moj) [reply migration/whereabouts-api.sh:26]: Both `secret` and `secrets` do the same 🤷‍♂️

**Pull request merged**